incubator/ark-server/templates/hook_delete.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: delete-ark-config
+  namespace: {{ .Values.namespace.backup.name }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: delete-ark-config
+      namespace: {{ .Values.namespace.backup.name }}
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: delete-ark-config
```

Comment:
Make list indentation consistent. This should indent one more level. Apply everywhere.